pkg/capture/capture.go

```diff
@@ -424,6 +424,12 @@ func closing(c *Capture) stateFn {
 	close(c.cmdChan)
 	c.closed = true
 
+	// free resources of the capture handle
+	if err := c.captureHandle.Free(); err != nil {
```

Comment:
@els0r Four-eyes principle: Is it correct to assume that the processing routine has finished at this point, or could it still be happening that `process()` is running? I doubt it (because at this point everything needs to have been rotated and hence left the blocking state) but I'm not 100% sure...

Reply:
Pretty sure that's the case. Will provide more meaningful feedback once in front of proper screen.

Reply:
It is the case, although not immediately visible: the only way to reach state `closing` is via the capture context being cancelled. That only happens _after_ the rotation has been completed (see line 240 and onwards in `main` function, or `Update` of the capture manager).

Reply:
Excellent, thanks for cross-checking! :clap: